### PR TITLE
Feat: add admin dashboard complete

### LINKED ISSUE #20: 19. Admin Dashboard
As an admin,
When I visit the admin dashboard (/admin)
Then I see a header indicating that I am on the admin dashboard

### PR DESCRIPTION
**Pull request by JorjaF**
__1__ Wrote Tests ____ Implemented ____ Reviewed

Neccesary checkmarks:

    [x] All Tests are Passing

    [x] The code will run locally

Type of change

    [x] New feature
    [] Bug Fix

Implements/Fixes:

    description closes #20 added admin dashboard index

Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

    [x] My code has no unused/commented out code
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

(REQUIRED)Please Include a link to a gif of your feelings about this branch

Link: https://steamcommunity.com/sharedfiles/filedetails/?id=590665774

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  namespace :admin do
```
Comment (z-fitch) [inline config/routes.rb:6]: good use of a namespace!

**Review by z-fitch — APPROVED** [review]

File: app/views/admin/dashboard/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div id="dashboard-header">
```
Comment (ethanrossblack) [inline app/views/admin/dashboard/index.html.erb]: do you want to change the name of the div id to something more like

`<div id="admin-header">` or `<div id="admin-dashboard-header">`

The reason is that we also have a Merchants dashboard header that is different than the Admin dashboard header.

File: spec/features/admin/dashboard_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+# As an admin,
+# When I visit the admin dashboard (/admin)
+# Then I see a header indicating that I am on the admin dashboard
+  describe "When I visit the admin dashboard (/admin)" do
+    it "I see a header indicating that I am on the admin dashboard" do
+
+      visit admin_dashboard_index_path
+
+      expect(page).to have_content("Admin Dashboard")
```
Comment (ethanrossblack) [inline spec/features/admin/dashboard_spec.rb]: Do you want to put this inside a "within" block?

**Review by ethanrossblack — APPROVED** [review]
nice advanced routing

**Review by Daniel-Gallagher92 — APPROVED** [review]
Within block put in!

**Pull request merged**